source/includes/_node.md

```diff
@@ -2,106 +2,216 @@
 
 ## JSON-RPC Requests 
 
-```shell
-
-```
-
-```javascript
-
-```
+Route for JSON-RPC requests, most of which mimic the bitcoind RPC calls completely.
 
-> The above command returns JSON structured like this:
+### HTTP Request
+`POST /`
```

Comment:
I think this needs separate file. Because there many RPC Requests that needs to be documented. So I left it out for now